build/NuGetPackages/Microsoft.Build.Runtime.nuspec

```diff
@@ -19,12 +19,14 @@
         <dependency id="Microsoft.Build.Framework" version="[$version$]" />
         <dependency id="Microsoft.Build.Tasks.Core" version="[$version$]" />
         <dependency id="Microsoft.Build.Utilities.Core" version="[$version$]" />
+        <dependency id="System.Runtime.InteropServices.RuntimeInformation" version="4.0.0" />
```

Comment:
MSBuild.exe depends on RuntimeInformation via the MSBuildEnvironmentHelper, which is compiles into MSBuild.exe. Because of that, I thought it might be better to make the dependency explicit.
On the other hand, everything used by msbuild.exe package flows from the other packages. so I'll remove it for consistency's sake :)